### DISCUSSION TITLE
Feature proposal.

### DISCUSSION THREAD

**Discussion by Sky-Evil** [TetraTheta/HardPlus#15] [Ideas]
Hello, could you implement a special health regeneration function, for example, so that certain players could restore hunger only with golden apples, or regeneration and healing potions, or beacons?

That is, in regular hardcore, as I understand it, it works like this: health restoration from satiety is simply disabled, and only restoration from the regeneration effect or instant healing remains. And this means that any sources of regeneration will restore the player's health (golden apple, potion, beacon).

This is also not bad, but it would be even better if it were possible to configure this separately, for example, so that certain players could restore health only with golden apples, or only with an instant healing potion, or only under a beacon.

Also, it would be nice if you left your contacts in the discord, or made your own discord server. I think your plugin is very promising.

**Comment by TetraTheta** [comment]
@Sky-Evil

The idea of restricting health regen by hunger satiation is doable. Thank you for the idea!

Since HardPlus is the only plugin that I've made for the public, I think running the Discord server only for this would require too much effort. I'll just use GitHub Discussion for community stuff until I create more Minecraft plugins and want to run a community from them. Thank you for the suggestion!

**Comment by TetraTheta** [comment]
New version of HardPlus is published. Wiki is also updated.
